prow/README.md

```diff
@@ -365,8 +367,10 @@ You may also have to create a keypair with the Metal3 CI ssh key.
 
    ```bash
    kind create cluster
-   clusterctl init --infrastructure=openstack:v0.11.1 --core=cluster-api:v1.8.5 \
-      --bootstrap=kubeadm:v1.8.5 --control-plane=kubeadm:v1.8.5
+   kubectl apply -k capi
+   # NOTE! You WILL need to apply multiple times before it is successful.
+   # This is because CRDs and webhooks must be in place before other
+   # resources can be added.
```

Comment:
Note this specifically!
This is an opinionated way of working. I put all of it in one kustomization to keep things simple.
However, that kustomization can never be applied in one go. It will *always* fail the first time you apply, because you cannot create a Certificate or CoreProvider until the CRDs and webhooks are in place.
So the question is, would you rather have a script that does this or is it good like this? Or do you want something different?

To make it crystal clear, the way I would use this if I have to deploy prow from scratch is like this:

1. kind create cluster
2. kubectl apply -k capi <- This would fail
3. Wait a few seconds
4. kubectl apply -k capi <- This could potentially succeed but will probably fail
5. Wait a few seconds
6. kubectl apply -k capi <- This would most likely succeed
7. Continue to the next step

Reply:
That is not something you do all the time, so documenting it clearly (like above comment) is enough IMO.